### ISSUE TITLE
create a tag and a package in release page

### ISSUE THREAD

**Issue by Sispheor** [kumina/openvpn_exporter#9]
Hi, could you tag a version of your exporter and create a release page with a pre compiled binary in a tarball? Like the [node exporter does](https://github.com/prometheus/node_exporter/releases).

This would be helpful to create tools like Ansible wrapper that deploy automatically the exporter.

Thanks a lot !

**Comment by EdSchouten** [comment]
Hello Nicolas,

We personally don't care that much about tagging/packaging these, as we at Kumina just build a binary from master, test it and deploy it internally.

We wrote these exporters simply because there weren't any (good) alternatives out there. We have published them on GitHub, but make little to no guarantees about what we can facilitate for the community. That said, it's very important that efforts to provide good exporters don't end up getting scattered all over GitHub.

Long story short: Kumina won't aim to do regular releases, tagging, packaging, automated builds, automated tests, etc. but if someone from the community wants to step up and do the lifting: we would very much applaud that. Would you be interested in doing this kind of work? If so, let me know and I'll add you to the project.

Ed

**Comment by Sispheor** [comment]
I wont have time to setup a complete CI chain but I can for sure help to maintain this little script when I can.
I'll keep an eye on the project.

Thank you for your time.
